crates/node/src/shell/finalize_block.rs

```diff
@@ -1850,7 +1851,10 @@ mod test_finalize_block {
             let req = FinalizeBlock {
                 txs,
                 proposer_address: proposer_address.clone(),
-                votes: votes.clone(),
+                decided_last_commit: tendermint::abci::types::CommitInfo {
+                    round: 0u8.into(),
```

Comment:
rather than passing 0 here, shouldn't we be passing the round value contained in the request?
https://docs.rs/tendermint/0.37.0/tendermint/abci/request/struct.FinalizeBlock.html#structfield.decided_last_commit

Reply:
This is just a unit test, we don't have a real request from which we can extract that information